packages/cli/src/databases/entities/SharedCredentials.ts

```diff
@@ -10,7 +10,7 @@ export class SharedCredentials extends WithTimestamps {
 	role: Role;
 
 	@Column()
-	roleId: string;
+	roleId: string; // @TODO: This should be string but is currently number
```

Comment:
are we planning to change this column type? if not, maybe we should remove this `TODO`.

Reply:
It's something I noticed but out of scope here. Or you mean we should do it in this PR?

Reply:
I'm just wondering if there is any plan to change `roleId` to `string` in the DB, and if there are any valid reasons to do so? 
also, we might be able to drop this field all together, and use `role.id`. 
either way, I personally don't find value in this `TODO` comment. 

maybe @krynble has an opinion.

Reply:
Synced with Omar, we'll remove the comments and add a transformer to stringify the ID as a fast follow. The original motivation for this was that I found [this check without `toString()` wasn't working](https://github.com/n8n-io/n8n/pull/6892/files#diff-9605a17e2cb66487c428992ac386a0b09df467ac6ef84ef146f3d04fb3eabc24R46).